lib/tensor.ts

```diff
@@ -31,16 +33,13 @@ export declare namespace Tensor {
   export type FloatType = Tensor.DataTypeMap['float32']|Tensor.DataTypeMap['float64'];
   export type NumberType = BooleanType|IntegerType|FloatType;
 
-  export interface Id {
-    // this field helps typescript to perform type check, comparing to use `Id` as an alias of object.
-    _tensorDataId_unused?: never;
-  }
+  export type Id = number;
```

Comment:
personally I feel a unique number is better as the ID than the tensor object. I feel it is much easier for me to debug the texture cache with a number as the key. 

This is more like my preliminary modification. Many things like multi-threading would affect this implementation. I think we discussed about adopting a more debug-friendly type for the ID. Depending on the scope of work, I'll either add it here, or open a new PR for this.

Reply:
+1 for this change, using an empty object as a key provides no traceability. Consider using a UUID or GUID generator library for uniqueness and thread safety(althoguh i dont think that would be necessary since browser tabs are mainly single-threaded).

Reply:
The measurement shows the perf diff between using number and UUID string is negligible. Updated using UUID. Thanks.